### DISCUSSION TITLE
How does this config should look like?

### DISCUSSION THREAD

**Discussion by Andrey22l** [confluentinc/confluent-kafka-go#741] [General]
Hello, 
How does this config  `sasl.oauthbearer.config`  should look like? Could not find an example
https://github.com/confluentinc/confluent-kafka-go/blob/1d5067b0f4f88f4167cb66a78ab4c4f3ec3b5786/examples/oauthbearer_example/oauthbearer_example.go#L132

**Comment by deshmukhrajvardhan** [comment]
Here is the only example i could find.
https://github.com/confluentinc/confluent-kafka-go/blob/194896a278506e9ac36e48d54abd123f0dcb3800/kafka/consumer_test.go#L324
